uaclient/messages/__init__.py

```diff
@@ -119,6 +120,11 @@ class TxtColor:
     "Cannot install package {package} version {version}"
 )
 
+WARN_PACKAGES_REMOVAL = "The following package(s) will be REMOVED:"
+WARN_PACKAGES_REINSTALL = (
+    "The following packages will be reinstalled from the archive:"
+)
```

Comment:
Either both of these should use "(s)" or they should be pluralized

Reply:
(s) for the easy go